app/javascript/crayons/Modal/__stories__/Modal.stories.jsx

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs/react';
+import { Modal, Button } from '@crayons';
+
+import '../../storybook-utiltiies/designSystem.scss';
```

Comment:
Noticed this folder name typo; probably better to leave this for a different PR 🙃

Reply:
cc @nickytonline

Reply:
Fixed it in #7671